src/libs/actions/BankAccounts.js

```diff
@@ -71,12 +77,22 @@ function addPersonalBankAccount(account, password, plaidLinkToken) {
         }),
     })
         .then((response) => {
-            if (response.jsonCode !== 200) {
-                alert('There was a problem adding this bank account.');
+            if (response.jsonCode === 200) {
+                PaymentMethods.getPaymentMethods()
+                    .then(() => {
+                        Navigation.goBack();
+                        Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: false});
```

Comment:
Do we want success growls here?

Reply:
Dang yea that would probably be a nice touch. It's pretty strange that we just kick you back to the payments page now that I think about it. But it works for now.